src/CarbonAware.WebApi/test/integrationTests/mocks/WattTimeServerMocks.cs

```diff
@@ -0,0 +1,140 @@
+            Frequency = 300,
+            Market = "mkt",
+            PointTime = testDataPointOffset,
+            Value = 999.99F,
+            Version = "1.0"
+        };
+
+        private static readonly List<GridEmissionDataPoint> defaultDataList = new() { defaultDataPoint };
+
+        private static readonly List<Forecast> defaultForecastList = new()
+        {
+            new Forecast()
+            {
+                GeneratedAt = testDataPointOffset,
+                ForecastData = new List<GridEmissionDataPoint>()
+                    {
+                        new GridEmissionDataPoint()
+                        {
+                            BalancingAuthorityAbbreviation = testBA,
+                            PointTime = testDataPointOffset,
+                            Value = 999.99F,
+                            Version = "1.0"
+                        }
+                    }
+            }
+        };
+
+        private static readonly BalancingAuthority defaultBalancingAuthority = new()
+        {
+            Id = 12345,
+            Abbreviation = testBA,
+            Name = "Test Balancing Authority"
+        };
+
+        private static readonly LoginResult defaultLoginResult = new() { Token = "myDefaultToken123" };
+
+        /// <summary>
+        /// Getter for the default data point used in the 
+        /// </summary>
+        public static GridEmissionDataPoint GetDefaultEmissionsDataPoint() => new()
+        {
+            BalancingAuthorityAbbreviation = defaultDataPoint.BalancingAuthorityAbbreviation,
+            Datatype = defaultDataPoint.Datatype,
+            Frequency = defaultDataPoint.Frequency,
+            Market = defaultDataPoint.Market,
+            PointTime = defaultDataPoint.PointTime,
+            Value = defaultDataPoint.Value,
+            Version = defaultDataPoint.Version
+        };
+
+        /// <summary>
+        /// Setup the mock server for watttime calls
+        /// </summary>
+        public static void WattTimeServerSetupMocks(this WireMockServer server, List<GridEmissionDataPoint>? dataMock = null, List<Forecast>? forecastMock = null, BalancingAuthority? baMock = null, LoginResult? loginMock = null)
+        {
+            SetupBaMock(server, baMock);
+            SetupLoginMock(server, loginMock);
+        }
+
+        /// <summary>
+        /// Helper function for setting up server response given a get request.
+        /// </summary>
+        /// <param name="server">Wire mock server to add mock to.</param>
+        /// <param name="path">String path server should respond to.</param>
+        /// <param name="statusCode">Status code server should respond with.</param>
+        /// <param name="contentType">Content type server should return.</param>
+        /// <param name="body">Response body from the request.</param>
+        private static void SetupResponseGivenGetRequest(this WireMockServer server, string path, HttpStatusCode statusCode, string contentType, string body)
```

Comment:
Love the flexibility here.  Consider adding `HttpStatusCode.OK, MediaTypeNames.Application.Json` as defaults since that is going to be a very common mock for this case.